### PR TITLE
RF 11.02 | Views > New View > Rename view

### PR DESCRIPTION
**Pull request by Turii**
https://trello.com/c/idxDqbGf

### CODE REVIEW

File: src/test/java/school/redrover/NewViewTest.java [hunk 1]
```diff
@@ -22,16 +20,16 @@ public class NewViewTest extends BaseTest {
...
-        getDriver().findElement(GO_TO_DASHBOARD_BUTTON).click();
+    private void createNewFreestyleProjectFromMyViewsPage(String projectName) {
+        new MainPage(getDriver())
+                .clickMyViewsSideMenuLink()
+                .clickNewItem()
+                .enterItemName(projectName)
+                .selectFreestyleProjectAndOk()
+                .clickSaveButton()
+                .clickDashboard();
+
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/NewViewTest.java:32]: blank line

File: src/test/java/school/redrover/model/ActiveViewPage.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+public class ActiveViewPage extends BaseMainHeaderPage<ActiveViewPage> {
+    public ActiveViewPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public String getActiveViewName() {
+
+        return TestUtils.getText(this, getDriver().findElement(By.xpath("//div[@class = 'tab active']")));
+    }
+
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/model/ActiveViewPage.java:17]: blank line

File: src/test/java/school/redrover/model/MyViewsPage.java [hunk 3]
```diff
@@ -88,6 +89,33 @@ public String getStatusMessageText() {
...
+
+    public MyViewsPage editMyViewNameAndClickSubmitButton(String editedMyViewName) {
+        TestUtils.click(this, getDriver().findElement(By.xpath("//a[contains(@href, '/configure')]")));
+        TestUtils.sendTextToInput(this, getDriver().findElement(By.xpath("//input[@name = 'name']")), editedMyViewName);
+        TestUtils.click(this, getDriver().findElement(By.xpath("//button[@name = 'Submit']")));
+
+        return this;
+    }
+
+
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/model/MyViewsPage.java:118]: two blank lines

**Review by evanmaiqa — APPROVED** [review]

**Review by DudeAlex — APPROVED** [review]
LGTM

**Pull request merged**